public/static/js/spawned_hosts.js

```diff
@@ -105,6 +105,15 @@ mciModule.controller('SpawnedHostsCtrl', ['$scope', '$window', '$timeout', '$q',
       $scope.currentResourceType = rtype;
     }
 
+    $scope.isHomeVolume = function(volume) {
+      for (var i = 0; i < $scope.hosts.length; i++) {
+        if ($scope.hosts[i].homeVolumeID === volume.volume_id) {
```

Comment:
does this need to be `$scope.hosts[i].home_volume_id`?

Reply:
oops yes